### DISCUSSION TITLE
GNSS not working on QField Windows V.3.1.9

### DISCUSSION THREAD

**Discussion by MappingMiles** [opengisch/QField#5017] [Q&A]
Hi,

I am using QField on Panasonic Toughpad FZ-G1. I am trying to connect a Hemisphere A222 GNSS Antenna as an external positioning device. I can find the com port in QField, and know the device works as it shows in other programs. But in QField it shows no location information.

Does anyone have any suggestions? I have tried various Baud rates, using NMEA GGA. 
TIA

**Comment by nirvn** [comment]
@MappingMiles , does it work in QGIS?

**Reply by MappingMiles** [reply]
Hi @nirvn , Yes it works in QGIS desktop (apologies had to download and test as this devie didn't have the desktop version). I am investigating using QField as an in field alternative as it is easier to use on a small screen!

**Reply by bettellam** [reply]
@MappingMiles  Try enabling **NMEA RMC** messaging!
https://receiverhelp.trimble.com/alloy-gnss/en-us/NMEA-0183messages_RMC.html
